app/lib/core/services/database/firebase_database_interface.dart

```diff
@@ -1,124 +1,105 @@
-
+  @override
   Future<FirestoreUser> getUser(String userId) async => _firestore
       .collection('users')
       .doc(userId)
       .get()
       .then((DocumentSnapshot snapshot) =>
           FirestoreUser.fromJSON(snapshot.data()));
 
-  Future<void> newRecord(AnimalTransportRecord newRecord) async =>
-      _firestore.collection('atr').add(newRecord.toJSON());
-
-  Future<bool> checkConnectivity() async {
-    var connectivityResult = await (Connectivity().checkConnectivity());
-    if (connectivityResult == ConnectivityResult.mobile) {
-      // I am connected to a mobile network.
-      return true;
-    } else if (connectivityResult == ConnectivityResult.wifi) {
-      // I am connected to a wifi network.
-      return true;
-    } else
-      return false;
-  }
-
-  Future<MySqlConnection> connectRemote() async {
-    final remoteConnect = await MySqlConnection.connect(ConnectionSettings(
-        host: 'localhost',
-        port: 0000,
-        user: 'root',
-        db: 'remote_database',
-        password: "capstone"));
-    return remoteConnect;
-  }
-
-  Future<Results> getRecord(User user) async {
-    dbConnect = await connectRemote();
-    var recentRecord = await dbConnect.query('''DATABASE QUERY''');
-    dbConnect.close();
-    return recentRecord;
-  }
-
-  Future<dynamic> retrieveUserRecords(User user) async {
-    dbConnect = await connectRemote();
-    var formList = await dbConnect.query('''DATABASE QUERY''');
-    dbConnect.close();
-    return formList;
-  }
-
-  Future<dynamic> updateTranInfo() async {
-    dbConnect = await connectRemote();
-    var updateTransporter = await dbConnect.query('''DATABASE QUERY''');
-    dbConnect.close();
-    return updateTransporter;
-  }
-
-  Future<dynamic> updateShipperInfo() async {
-    dbConnect = await connectRemote();
-    var updateShipper = await dbConnect.query('''DATABASE QUERY''');
-    dbConnect.close();
-    return updateShipper;
-  }
-
-  Future<dynamic> updateFwr() async {
-    dbConnect = await connectRemote();
-    var updateFwr = await dbConnect.query('''DATABASE QUERY''');
-    dbConnect.close();
-    return updateFwr;
-  }
-
-  Future<dynamic> updateVehicleLoadingInfo() async {
-    dbConnect = await connectRemote();
-    var updateVehicle = await dbConnect.query('''DATABASE QUERY''');
-    dbConnect.close();
-    return updateVehicle;
-  }
-
-  Future<dynamic> updateDeliveryInfo() async {
-    dbConnect = await connectRemote();
-    var updateDelivery = await dbConnect.query('''DATABASE QUERY''');
-    dbConnect.close();
-    return updateDelivery;
-  }
+  @override
+  Future<void> newUser(FirestoreUser newUser) async =>
+      _firestore.collection('users').doc(newUser.userId).set(newUser.toJSON());
 
-  Future<dynamic> updateAckInfo() async {
-    dbConnect = await connectRemote();
-    var updateAck = await dbConnect.query('''DATABASE QUERY''');
-    dbConnect.close();
-    return updateAck;
+  @override
+  Future getRecord(User user) {
```

Comment:
I imagine @SanaKhan1997 is working on the group of functions for fetching in-progress ATRs, ATRs that match search critera, ATRs from the last month, etc.?